tracer/test/Datadog.Trace.Tests/Telemetry/Metrics/common_metrics.json

```diff
@@ -728,7 +728,8 @@
     "rasp.rule.eval": {
       "tags": [
         "waf_version",
-        "rule_type"
```

Comment:
The values has been updated in the dd-go repo common_metrics file as well

Reply:
Yes, it's a new flag that was introduced to distinguish between command injection and shell injection vulnerabilities and it's defined for all the libraries in the RFC.